pkg/testutil/well_known_generation_test.go

```diff
@@ -72,7 +72,7 @@ func createWellKnownDIDConfiguration(tt *testing.T, didResponse *did.CreateDIDRe
 	err = builder.SetCredentialSubject(subject)
 	assert.NoError(tt, err)
 
-	err = builder.AddContext("https://identity.foundation/.well-known/did-configuration/v1")
+	err = builder.AddContext("https://tbd.website/.well-known/did-configuration/v1")
```

Comment:
test util is for integration tests only? if not we should rethink this

Reply:
testutil is currently being skipped right now. I'm of the opinion that we should remove, and add some of this functionality to a CLI.